packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -134,6 +134,19 @@ export interface IntegrationProps {
   readonly options?: IntegrationOptions;
 }
 
+/**
+ * Result of binding an Integration to the Method
+ */
+export interface IntegrationConfig {
+  /**
+   * This value is included in computing the Deployment's fingerprint. When the fingerprint
+   * changes a new deployment is triggered.
+   * This property should carry a different value when the Deployment needs to be refreshed.
+   * @default - deployments are not triggered for any change of this integration.
```

Comment:
Add a note that it can be an arbitrary string and does not have to be a hash.

Reply:
Switched to index type <string, string> so the user doesn't have to do JSON.stringify() unless they're adding a nested struct. The input to the hash gen will also be a bit nicer without the serialized objects.